src/shapes/box2d_shape.h

```diff
@@ -31,7 +36,9 @@ class Box2DShape {
 	virtual Variant get_data() const = 0;
 
 	virtual int get_b2Shape_count(bool is_static) const = 0;
-	virtual b2Shape *get_transformed_b2Shape(int p_index, const Transform2D &p_transform, bool one_way, bool is_static) = 0;
```

Comment:
The transform also isn't part of the resource (each body can transform the shape differently), which explains why I had the method like this.

Reply:
Perfect, wasn't sure of that. Updating that.